### ISSUE TITLE
Design: Scope

### ISSUE THREAD

**Issue by BrainBlasted** [BrainBlasted/Trumpet#9]
This issue is to work out what exactly the scope of the application should be as a Mastodon client.
Using the guidelines laid by the GUADEC video in #8, I came up with the following Must-haves and Won't-haves:

## Must-Haves

* Sending and receiving toots
* Content and Sensitive media warnings
* Timeline viewing
* Custom  (pinned) timelines
* Notifications
* Toot conversation view
* Replying
* Direct Messages
* Mute & Block Management
* Multi-account (Working in cli version)

## Won't-Haves

* Tracking
* Status text search (for privacy and anti-harassment reasons)
* Encryption (A chat protocol is suggested for private communications)

Community input on this is encouraged.

**Comment by rscmbbng** [comment]
Multiple account support would be great!

**Comment by BrainBlasted** [comment]
That is definitely something to include.

**Comment by ryliejamesthomas** [comment]
Yeah, that's a big must-have for me too! Especially multiple accounts per-instance, which some clients seem to lack.

Another thing I'd really like would be for it to be window-size aware, so I can keep it in a narrow window on the edge of the screen and it still be usable.

**Comment by BrainBlasted** [comment]
I have moved the project to the GNOME GitLab. The new url is https://gitlab.gnome.org/BrainBlasted/Trumpet
